src/Symfony/Bundle/WebProfilerBundle/Tests/Twig/WebProfilerExtensionTest.php

```diff
@@ -23,7 +24,7 @@ class WebProfilerExtensionTest extends TestCase
      */
     public function testDumpHeaderIsDisplayed(string $message, array $context, bool $dump1HasHeader, bool $dump2HasHeader)
     {
-        $twigEnvironment = $this->mockTwigEnvironment();
```

Comment:
the unused private method `mockTwigEnvironment` should be removed

Reply:
oh indeed, good catch